### ISSUE TITLE
[blog]: several images are missing alt text.

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#663]

**Comment by jankeromnes** [comment]
Side note: On its own, this issue is not very helpful (no issue description). Adding one or two examples of images that don't have an alt text in the issue description would help.
